pkg/cri/cri.go

```diff
@@ -0,0 +1,66 @@
+package cri
+
+import (
+	"context"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	k8sutil "k8s.io/kubernetes/pkg/kubelet/util"
+)
+
+const (
+	maxMsgSize   = 1024 * 1024 * 16
+	socketPrefix = "unix://"
```

Comment:
Oh, this came from `pkg/agent/containerd/config_linux.go`. You probably need to split some of this into `cri_linux.go` and `cri_windows.go` with any platform-specific stuff (path handling mostly) in the correct file. You also need to make the corresponding package moves in ``pkg/agent/containerd/config_windows.go`.